smart_campus/app/views.py

```diff
@@ -618,3 +639,390 @@ def reward_add_page(request):
     }
 
     return render(request, 'app/reward_add_page.html', context)
+
+
+@login_required
+def manager_list_page(request):
+    # List the manager on the page
```

Comment:
Do not use `#` to add doc for a function.
Use `"""List the manager on the page"""`.
This should be fixed elsewhere as well

Reply:
Okay, I will fix it.
Can I ask why should not use ```#```?

Reply:
@Lee-W

Reply:
`#` is used as a comment.
It will not be executed at runtime.
But, `"""Document"""` will.
It can be called by using `__doc__`.

Note that if the comment is not for documenting this bode block (e.g., class, function, module), you should use `#` instead.

Reply:
Got it. ty.